api/proto/info/info.proto

```diff
@@ -65,6 +65,7 @@ enum ZInfoTypes {
   ZiContentTree = 8;
   ZiBlobList = 9;
   ZiAppInstMetaData = 10;
+  Zi_Edgeview = 11;
```

Comment:
We can ignore the yetus complaint here and follow the current naming scheme. However, the current one does not have an undescore in  the name so ZiEdgeview would be the consistent name.

Reply:
Changed. I originally was using 'ZiEdgeview', but after saw yetus error and change to with '_' and noticed yetus actually want them all capital. Ok, back to 'ZiEdgeview' now.